bootstrap.sh

```diff
@@ -70,7 +70,7 @@ function setup_package() {
       if [ "${SQLALCHEMY_VERSION}" = "latest" ]; then
         pip install "sqlalchemy" --upgrade
       else
-        pip install "sqlalchemy==${SQLALCHEMY_VERSION}"
+        pip install "sqlalchemy==${SQLALCHEMY_VERSION}" --pre
```

Comment:
Is the `--pre` still required?

Reply:
Thanks for spotting. I don't think so, so I've updated 71a7f0ad73 correspondingly.